ai-prompts/azure-cli.txt

```diff
@@ -33,6 +35,7 @@ Best practices
 
 Placeholders
   - Always use descriptive placeholders like `<project-id>`, `<region>`, `<vm-name>`, etc.
+  - If using placeholders, state: `Note: This command uses placeholders which are signaled with <>` 
```

Comment:
I thought about using 'PLACEHOLDER' but i really dont think its better than using <>, infact I think its more confusing, I think it makes more sense to stick with common practice and instead opt to explain it. WDYT? @rita-gorokhod